control-base/src/main/java/fi/nls/oskari/control/view/modifier/bundle/MapfullHandler.java

```diff
@@ -446,10 +447,9 @@ private static void appendMyPlacesLayers(final JSONArray layerList,
 
             JSONObject myPlaceLayer = null;
             if (plugins != null && plugins.toString().indexOf(PLUGIN_WFSVECTORLAYER) != -1) {
```

Comment:
I don't think we need to check the plugin anymore. Having it present or not doesn't really matter now. The frontend can either show this as a wfslayer or log a warning that it doesnt recognize a layer with type wfs.